hoomd/write/table.py

```diff
@@ -353,14 +351,14 @@ class Table(_InternalCustomWriter):
             the data from, defaults to standard out. The object must have write
             and flush methods and a mode attribute.
         header_sep (`str`, optional): String to use to separate names in
-            the logger's namespace, defaults to '.'. For example, if logging the
-            total energy of an `hoomd.md.pair.LJ` pair force object, the default
-            header would be ``md.pair.LJ.energy`` (assuming that
+            the logger's namespace, defaults to ``'.'``. For example, if logging
+            the total energy of an `hoomd.md.pair.LJ` pair force object, the
+            default header would be ``md.pair.LJ.energy`` (assuming that
             ``max_header_len`` is not set).
         delimiter (`str`, optional): String used to separate elements in
-            the space delimitated file, defaults to ' '.
+            the space delimitated file, defaults to ``' '``.
         pretty (`bool`, optional): Flags whether to attempt to make output
-            prettier and easier to read, defaults to True. To make the ouput
+            prettier and easier to read, defaults to True. To make the output
             easier to read, the output will compromise on outputted precision
             for improved readability. In many cases, though the precision will
```

Comment:
"though" does not fit the flow of the sentence. Is that an error?
```suggestion
            for improved readability. In many cases, the precision will
```